CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 18.2.2 - [#742](https://github.com/openfisca/openfisca-france/pull/742)
```

Comment:
As you are not modifying the `openfisca_france` package, you don't need to publish a new version.